terminal/command.go

```diff
@@ -579,7 +607,14 @@ func printcontext(state *api.DebuggerState) error {
 		}
 	}
 
-	for i := l - 5; i <= l+5; i++ {
+	max := func(a, b int) int {
```

Comment:
Yeah, I'm not really sure what I was thinking with creating the separate function, I'll fix that.  But I don't think moving it up to line 603 is the right thing.  `l` is the current position in the file and required state for drawing the arrow, and so we don't want to change it.

I could use `line` instead, but it gets shadowed in the inner loop as a string.  I could fix that too, I guess it's just a question of how much cleanup you want me to do there.

Just inlining the function, the diff would be something like this:

```diff
diff --git i/terminal/command.go w/terminal/command.go
index e44d27e..1661e88 100644
--- i/terminal/command.go
+++ w/terminal/command.go
@@ -607,14 +607,12 @@ func printfile(filename string, line int) error {
                }
        }

-       max := func(a, b int) int {
-               if a > b {
-                       return a
-               }
-               return b
+       s := l - 5
+       if s < 0 {
+               s = 0
        }

-       for i := max(0, l-5); i <= l+5; i++ {
+       for i := s; i <= l+5; i++ {
                line, err := buf.ReadString('\n')
                if err != nil {
                        if err != io.EOF {
```